packages/loot-core/src/server/budget/types/handlers.d.ts

```diff
@@ -13,15 +13,13 @@ export interface BudgetHandlers {
 
   'budget/set-3month-avg': (...args: unknown[]) => Promise<unknown>;
 
+  'budget/set-n-month-avg': (...args: unknown[]) => Promise<unknown>;
+
   'budget/check-templates': () => Promise<Notification>;
 
-  'budget/apply-goal-template': (arg: {
-    month: string;
-  }) => Promise<Notification>;
+  'budget/apply-goal-template': (...args: unknown[]) => Promise<unknown>;
 
-  'budget/overwrite-goal-template': (arg: {
-    month: string;
-  }) => Promise<Notification>;
+  'budget/overwrite-goal-template': (...args: unknown[]) => Promise<unknown>;
 
```

Comment:
Were these changes intentional?

Reply:
These were giving me TS errors in the tests. I'm uninformed on how to handle these errors, but this change made them pass the test.  I added 'async' to both of those functions which caused the errors to come up.  Is there something I need to add to the function line for the test to pass?

Reply:
I just reverted the changes.  I don't see the errors now.